ts/src/latoken.ts

```diff
@@ -1095,20 +1095,17 @@ export default class latoken extends Exchange {
         let market = undefined;
         const isTrigger = this.safeValue2 (params, 'trigger', 'stop');
         params = this.omit (params, 'stop');
-        if (symbol === undefined) {
-            // privateGetAuthOrderActive doesn't work even though its listed at https://api.latoken.com/doc/v2/#tag/Order/operation/getMyActiveOrders
-            throw new ArgumentsRequired (this.id + ' fetchOpenOrders requires a symbol argument');
+        this.checkRequiredSymbol ('fetchOpenOrders', symbol);
```

Comment:
next time just tell me to use `checkRequiredSymbol`, it will help me get in the habit of using it